src/Control/Effect/FossaApiClient.hs

```diff
@@ -137,3 +139,6 @@ queueArchiveBuild = sendSimple . QueueArchiveBuild
 
 assertUserDefinedBinaries :: Has FossaApiClient sig m => IAT.UserDefinedAssertionMeta -> [Fingerprint Raw] -> m ()
 assertUserDefinedBinaries meta fprints = sendSimple (AssertUserDefinedBinaries meta fprints)
+
+assertRevisionBinaries :: Has FossaApiClient sig m => Locator -> [Fingerprint Raw] -> m ()
```

Comment:
[No action] too many methods (about ~4 now) in different modules called the `assertRevisionBinaries` 😄  This adds a bit of friction in readability, but this is convention anyways, so no action but wanted to denote this.

Reply:
I'm hoping we can mess with names later, and we'll get rig of the ones in `FossaV1` soon.